src/functionalTest/resources/responses/bulk-scan-a59-validation-output.json

```diff
@@ -0,0 +1,13 @@
+{
+  "status":"ERRORS",
+  "warnings":{
+    "items":[
+
+    ]
+  },
+  "errors":{
+    "items":[
+
+    ]
+  }
+}
```

Comment:
pls check this json is correct, status should be success([bulk-scan-a59-validation-output.json] for functional tests